File: codesignals/MakeArrayConsecutive.py
def MakeArrayConsecutive(statues):
   s=0
   statues.sort()

   for i in range(0,len(statues)-1):
       s+=statues[i+1]-statues[i]-1
   return s

if __name__== "__main__":
    test_statues = [6, 2, 3, 8]
    print(f"Number of additional statues needed: {MakeArrayConsecutive(test_statues)}")
    
    test_statues2 = [1, 3, 5]
    print(f"Number of additional statues needed: {MakeArrayConsecutive(test_statues2)}")

File: codesignals/MakeArrayConsecutive.java
package codesignals;
import java.util.Arrays;

public class MakeArrayConsecutive {

    public static void main(String[] args) {
        int[] statues = {6, 2, 3, 8};
        int result = makeArrayConsecutive(statues);
        System.out.println("Number of additional statues needed: " + result);
    }

    public static int makeArrayConsecutive(int[] statues) {
      
        Arrays.sort(statues);
        return statues[statues.length - 1] - statues[0] + 1 - statues.length;
    }
}